charts/kubewarden-defaults/values.schema.json

```diff
@@ -12,19 +12,35 @@
           "type": "integer"
```

Comment:
I'm not sure if you we should use integer. The PDB fields should allow [percentage](https://kubernetes.io/docs/tasks/run-application/configure-pdb/#rounding-logic-when-specifying-percentages) representation as well. Therefore, the more appropriate type may be `string` (as before) or [`number`](https://json-schema.org/understanding-json-schema/reference/numeric) .

Reply:
As we are using the same k8s type in our CRDs, I believe we must support percentage representation. I think it the best UX when users do not know the exact number of replicas (which can change over time) they will run

Reply:
You are totally right. I will change it back to string.

Reply:
Reverted to string, changed the XOR implementation. Please re-review.